lib/measured/measurable.rb

```diff
@@ -66,4 +68,20 @@ def name
     end
 
   end
+
+  private
+
+  def value_string
+    @value_string ||= begin
+      str = case value
+      when Rational
+        value.denominator == 1 ? value.numerator.to_s : value.to_f.to_s
+      when BigDecimal
+        value.to_s("F")
+      else
+        value.to_f.to_s
+      end
+      str.gsub(/\.0*\Z/, "")
```

Comment:
What do you think about coercing value to Float, then returning a formatted string using Ruby's `format` like this:

`format('%g', float_value)`

Thoughts?